packages/core/src/Stores/ui-store.js

```diff
@@ -162,6 +162,7 @@ export default class UIStore extends BaseStore {
     app_contents_scroll_ref = null;
     is_deriv_account_needed_modal_visible = false;
     is_exit_traders_hub_modal_visible = false;
+    is_ready_to_deposit_modal_visible = false;
```

Comment:
@thisyahlen-deriv Would you please add the type in `packages/stores/types.ts` 🙏🏻